client/components/command-palette/use-command-palette.tsx

```diff
@@ -82,7 +82,9 @@ export const useCommandPalette = ( {
 
 	// Sort sites in the nested commands to be consistent with site switcher and /sites page
 	const { sitesSorting } = useSitesSorting();
-	const sortedSites = useSitesListSorting( allSites, sitesSorting );
+	const sortedSites = useSitesListSorting( allSites, sitesSorting ).filter(
```

Comment:
Are these changes related to this PR and why are we adding them?

Reply:
Removed! it was debugging code. Thanks for catching it!  https://github.com/Automattic/wp-calypso/pull/85262/commits/77c895c2f0f4c0cc6f34c84f6ce0f2b338413e3c